test/fixtures/calendars.yml

```diff
@@ -1,11 +1,11 @@
 # Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
 
-one:
-  color: MyString
-  title: MyString
-  user: one
+michaels:
+  color: ffffff
+  title: Birthdays
+  user: michael
```

Comment:
What is that?

Reply:
Calendar model fixture, it should be used in tests.